CHANGELOG.md

```diff
@@ -171,6 +177,14 @@ for further information about branching and tagging conventions.
 #### Fixed
 - Not handling 'manage-asset-pair' operation details
 
+### Removed
+- `AssetRequestDetails` component
+- `SaleRequestDetails` component
+- `PreIssuanceCreateRequestRecord` wrapper
+- `SaleRequestRecord` wrapper
+- Update sale details JSON mock
+- Sale create request JSON mock
```

Comment:
these ones look like the list of "common records wrappers". anyway, they should be under "unreleased"